packages/ia-components/index.js

```diff
@@ -3,7 +3,10 @@
  */
 
 // live
-export { default as IAUXExampleComponent } from './live/example-react-component/index';
+//Next line fails webpack - complains Support for the experimental syntax 'classProperties' isn't currently enabled
```

Comment:
@mitra42 can you send me the webpack config that you are using? 

you may need to add `.jsx` into your bundling config.  i can help debug, that way we can drop the `.jsx` from the import statements.

Reply:
I have https://github.com/internetarchive/dweb-archive/blob/master/.babelrc and https://github.com/internetarchive/dweb-archive/blob/master/webpack.config.js 
edited recently so it does the webpack by @shreyansh23 and then I added the `js[x]`. 

The remaining problem is that for some reason it wasn't getting the import of DetailsFlags from DetailsActionButtons without it being DetailsFlags.jsx. I'll try and repeat the failing scenario to remind myself.